tasty-query/shared/src/main/scala/tastyquery/reader/Loaders.scala

```diff
@@ -19,7 +19,7 @@ private[tastyquery] object Loaders {
   class MissingTopLevelTasty(root: Loader.Root) extends Exception(s"Missing TASTy for ${root.fullName}")
 
   object Loader:
-    private[tastyquery] case class Root private[Loaders] (pkg: PackageSymbol, rootName: SimpleName):
+    private[Loaders] case class Root private[Loaders] (pkg: PackageSymbol, rootName: SimpleName):
       def packages: IterableOnce[PackageSymbol] =
```

Comment:
this method is now dead code - it used to be to validate that a tasty file's declared packages match the file name, but now it seems that check was dropped?